README.md

```diff
@@ -24,6 +24,12 @@ See [src/Linux/README.MD](src/Linux/README.MD).
 
 See [src/Windows/README.MD](src/Windows/README.MD).
 
+# Configuration
+
+The Azure-DCAP-Client library uses the following environment variables if set:
+
+* `AZDCAP_CACHE` - Represents the base directory where the library cache directory `.az-dcap-client` is created. The default value is `$HOME`.
+
```

Comment:
Can you also add a snippet about how you can configure the base url for the pck cache service here?

Reply:
Hello @msft-jonnadul!

Are you referring to these recently added environment variables ?
https://github.com/Microsoft/Azure-DCAP-Client/blob/1a82c45fe1150b377a9ff2451cde5c599242321e/src/dcap_provider.cpp#L26-L32

I thought about documenting them, but then I noticed the comment from @johnkord in the code.

However, I also think it's worth documenting them. What do you think @johnkord ?

Reply:
@msft-jonnadul I've also added some notes about these environment variables.

Please let me know if everything looks good to you now.